charts/rstudio-connect/NEWS.md

```diff
@@ -1,4 +1,12 @@
 # Changelog
+## 0.7.8
+
+- Bump Chronicle Agent to version 2024.09.0
+
+## 0.7.7
+
+- Add helm values for `pod.hostAliases` and `launcher.templateValues.pod.hostAliases`
+- Add helm values for `launcher.defaultInitContainer.resources`
```

Comment:
I'm a bit confused here - this change is already [on main](https://github.com/rstudio/helm/blob/main/charts/rstudio-connect/NEWS.md). Why is this included in the PR?